thirdparty/tinymce/tiny_mce_gzip.php

```diff
@@ -19,7 +19,7 @@
 	// Default settings
 	$tinyMCECompressor = new TinyMCE_Compressor(array(
 		// CUSTOM SilverStripe
-		'cache_dir' => TEMP_FOLDER
+		'cache_dir' => TEMP_FOLDER.'/tinymce-cache'

```

Comment:
It could be, but all I wanted was to invalidate cache for this instance, so you don't need to wait up to 30 days for the fix to take effect.